deploy/osd-ingress/router-infraNodes.patch.yaml

```
@@ -0,0 +1,8 @@
+
+apiVersion: operator.openshift.io/v1
+applyMode: AlwaysApply
+kind: IngressController
+name: default
```

```suggestion
name: default
namespace: openshift-ingress-operator
```

Not a cluster scope resource, have to specify namespace.